### PR TITLE
try out ruff

### PR DESCRIPTION
**Pull request by tarekziade**
### Description
Describe the proposed changes made in this PR.

### How was this PR tested?
Describe how you tested this PR.

### CODE REVIEW

File: nucliadb/setup.cfg [hunk 1]
```diff
@@ -1,19 +0,0 @@
...
-  E701
-  W504
-  F901
-  E252
-  W503
-  E203
-  BLK100
-  F541
-
-[tool:pytest]
```
Comment (jotare) [inline nucliadb/setup.cfg:16]: Not sure you need to remove this too

Comment (tarekziade) [reply nucliadb/setup.cfg:16]: it's moved in the toml file , see https://docs.pytest.org/en/7.3.x/reference/customize.html#pyproject-toml

**Review by jotare — COMMENTED** [review]
Could you also migrate gh actions (.github/workflows/...) and pre-commit hook?

File: nucliadb/setup.cfg (hunk 1 above)
Comment (tarekziade) [reply nucliadb/setup.cfg:16]: oh I forgot to commit it in the PR, forgot a `git add` :D sorry

**Review by lferran — APPROVED** [review]

**Pull request merged**